dialect_oracle.go

```diff
@@ -93,6 +94,10 @@ func (d OracleDialect) TruncateClause() string {
 	return "truncate"
 }
 
+func (d OracleDialect) SleepClause(s time.Duration) string {
+	return ""
+}
```

Comment:
If we were to use a separate `SleepDialect`, you could just skip implementing this method.